src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,33 @@
-export const TodoInfo = () => {};
+import cn from 'classnames';
+import { UserInfo } from '../UserInfo';
+import usersFromServer from '../../api/users';
+import { User, Todo } from '../../types';
```

Comment:
Not all files were commited
![2023-09-21_15h41_34](https://github.com/mate-academy/react_add-todo-form/assets/61904995/a75dde7b-fb8b-4972-856c-f491018a8d73)